e2e/cli-e2e/tests/__snapshots__/compare.e2e.test.ts.snap

```diff
@@ -0,0 +1,705 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`CLI compare > should compare report.json files and create report-diff.json 1`] = `
+{
+  "audits": {
+    "added": [],
+    "changed": [
+      {
+        "displayValues": {
+          "after": "62 %",
+          "before": "62 %",
+        },
+        "plugin": {
+          "slug": "coverage",
+          "title": "Code coverage",
+        },
+        "scores": {
+          "after": 0.617,
+          "before": 0.6211,
+          "diff": -0.0040999999999999925,
```

Comment:
I feel like the diff should also have 4 or so decimal digits - the highest level of precision based on `after` and `before` precision. 🤔

Reply:
I would lean towards keeping it simple and sticking to the JavaScript precision. The JSON isn't meant for display, it's the raw values. We will definitely round these values whenever we display it (e.g. for the Markdown diff).

Reply:
As long as different runs of E2E tests do not produce different values in snapshots and report false positives, it's fine by me.

Reply:
It isn't random, just hard to read.